app/views/editor/rich.html.erb

```diff
@@ -204,11 +204,19 @@
 
       <div class="ple-module-guide col-lg-3">
         <h2>4</h2>
+        
       </div>
-
-      <div class="ple-module-content col-lg-9">
-        <textarea id="text-input" class="ple-textarea form-control"><% if @node && @node.latest && @node.latest.body_rich %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
-      </div>
+      <% if @graphobject %>
+        <div class="ple-module-content col-lg-9">
+          <% x = "Power Tag: simple-data-grapher:i/" + @graphobject[0] + "/" + @graphobject[1] %>
```

Comment:
So this is where I'd prefer to try integrating this in the way that the original graph powertag was done - by detecting power tags that are actually in the body text itself: https://github.com/publiclab/PublicLab.Editor/search?q=powertags&unscoped_q=powertags

What you're trying to do here is make it possible to open the `/post` route while specifying a CSV file in the URL parameters, right? But can't we do that with the `body` parameter, as with this example?

https://publiclab.org/post?body=[notes:balloon-mapping]

So you'd just do something like:

https://publiclab.org/post?body=[simple-data-grapher:i/1/23] -- is that right?

Reply:
@Anupam-dagar you can take a look here